fl4health/server/model_merge_server.py

```diff
@@ -0,0 +1,104 @@
+import datetime
+import timeit
+from logging import DEBUG, INFO
+from typing import Optional, Tuple
+
+from flwr.common.logger import log
+from flwr.server.client_manager import ClientManager
+from flwr.server.history import History
+from flwr.server.server import Server
+from flwr.server.strategy import Strategy
+
+from fl4health.reporting.metrics import MetricsReporter
+from fl4health.strategies.model_merge_strategy import ModelMergeStrategy
+
+
+class ModelMergeServer(Server):
+    def __init__(
+        self,
+        *,
+        client_manager: ClientManager,
+        strategy: Optional[Strategy] = None,
+        metrics_reporter: Optional[MetricsReporter] = None,
+    ) -> None:
+        """
+        ModelMergeServer provides functionality to fetch client weights, perform a simple average,
+            redistirbute to clients for evaluation. Optionally can perform server side evaluation as well.
+        Args:
+            client_manager (ClientManager): Determines the mechanism by which clients are sampled by the server, if
+                they are to be sampled at all.
+            strategy (Optional[Strategy], optional): The aggregation strategy to be used by the server to handle.
+                client updates and other information potentially sent by the participating clients. If None the
```

Comment:
In this case, I don't think we'll be receiving other information? Maybe I'm wrong though.